lib/location/cloud_service/cloud_service_nrf.c

```diff
@@ -115,7 +115,8 @@ int cloud_service_nrf_pos_get(
 #endif
 	const struct nrf_cloud_rest_location_request loc_req = {
 		.cell_info = params->cell_data,
-		.wifi_info = params->wifi_data
+		.wifi_info = params->wifi_data,
+		.config = NULL
```

Comment:
This is not strictly necessary, because when you initialize a struct like this, rest of the fields are automatically initialized to zero.